platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/LoggerImpl.kt

```diff
@@ -529,6 +528,20 @@ internal class LoggerImpl(
             errorHandler.handleError("Couldn't start JankStatsMonitor", IllegalArgumentException("Invalid application provided"))
         }
     }
+
+    private fun Duration.writeSDKStartLog() {
```

Comment:
it looks awkward to me for this logic to be an extension method of Duration? wouldn't a simple plain method suffice? not everything has to be an extension method 😄